var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -19,6 +19,8 @@ class Mumps(Package):
     variant('float', default=True, description='Activate the compilation of smumps')
     variant('complex', default=True, description='Activate the compilation of cmumps and/or zmumps')
     variant('idx64', default=False, description='Use int64_t/integer*8 as default index type')
+    variant('tests', default=False, description='Run small MPI unit tests')
```

Comment:
The problem with adding and removing variants is that they affect the spec hash. So when you add/remove a variant, you need to reinstall the package if you want to link against it. If you would like to add the ability to test packages, then you should do it right and add some extra functionality to Spack. I'm willing to help with this.